src/core_modules/capture-core/components/Pages/Search/SearchPage.component.js

```diff
@@ -185,10 +190,20 @@ const Index = ({
                 }
 
                 {
-                    searchStatus === searchPageStatus.ERROR &&
-                    <h1 style={{ color: 'red' }}>{searchResultsErrorMessage}</h1>
+                    searchStatus === searchPageStatus.SHOW_RESULTS &&
+                    <h3>
+                        Your search has given results. At this point the results are stored.
+                        Number of results: {searchResults && searchResults.length}
+                    </h3>
+
                 }
 
+                {
+                    searchStatus === searchPageStatus.ERROR &&
```

Comment:
This is also needs to be worked out better. I wanted to ask where I can find some other examples of error messages so that we can have the same use and feel here.

Reply:
I think something like: "An error occurred while performing the search. Please see the console log for details".

Reply:
Talked with Joe about it. I have also created a ticket that is dedicated in adding better text in the places missing. 
The ticket: https://jira.dhis2.org/browse/DHIS2-9140

Joe suggested something between the lines "There is a problem with this search, please change the search terms or try again later." 

We can go with something that combines both such as "There is a problem with this search, please change the search terms or try again later. For more details open the Console tab of the Developer tools"

Reply:
What you're suggesting sounds good to me 👍